### PR TITLE
pass start at time to stop invoke

### LINKED ISSUE #396: "start --at" fails when stopping a current project
I'm already running a project, and I want to start a new one but backdate the transition, so I run:

```
watson start New_Project +tagged --at "16:14"
```

and it returns:

```
Stopping project Old_Project [tagged], started 37 minutes ago and stopped just now. (id: 19eb6f0)
Error: Task cannot start before the previous task ends.
```

So it's stopped the current project at the current time, but can't start the new one because then the who projects would overlap.

### PR DESCRIPTION
**Pull request by EdgyEdgemond**
closes #396

### CODE REVIEW

**Comment by jmaupetit** [comment]
Thank you for this @EdgyEdgemond :pray: Can you add a test for this fix please?

**Comment by EdgyEdgemond** [comment]
sure thing, was trying to find an existing start/stop test, hadn't located it yet

File: tests/test_cli.py [hunk 1]
```diff
@@ -196,6 +196,24 @@ def test_start_valid_time(runner, watson, mocker, at_dt):
...
+    start_dt = datetime(2019, 4, 10, 15, 0, 0, tzinfo=local_tz_info())
+    arrow.arrow.dt_datetime.now.return_value = start_dt
+    runner.invoke(cli.start, ['a-project', '--at', "14:10"], obj=watson)
+
+    result = runner.invoke(
+        cli.start,
+        ['b-project', '--at', "14:15"],
+        obj=watson,
+    )
+    assert result.exit_code == 0, result.stdout
```
Comment (jmaupetit) [inline tests/test_cli.py:214]: Can you also check that a frame has been added to the `frames` file and current session has been created?

**Review by jmaupetit — APPROVED** [review]
Perfect, thanks!

**Pull request merged**